client/src/components/ProductDetails/Checkout/AddToCart.js

```diff
@@ -1,15 +1,31 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import css from './Checkout.css';
 
-const AddToCart = ({ onMouseMove }) => (
-  <div className={css.addToCart} id="addToCart" onMouseMove={onMouseMove}>
-    <button type="submit" value="addToBag" className={css.addToCartButton}>
-      <span className="fa fa-shopping-bag" />
-      &nbsp;
-      ADD TO BAG
-    </button>
-  </div>
-);
+const AddToCart = ({ onMouseMove }) => {
+  const [added, add] = React.useState(false);
+  return (
+    <div role="button" className={css.addToCart} id="addToCart" onMouseMove={onMouseMove} onClick={() => { add(true); }} onKeyPress={() => {}} tabIndex={0}>
+      {added ? (
+        <button type="submit" value="addToBag" className={css.addToCartButton}>
+          <span className="fa fa-shopping-bag" />
+          &nbsp;
+          ADDED TO BAG
+        </button>
+      ) : (
+        <button type="submit" value="addToBag" className={css.addToCartButton}>
+          <span className="fa fa-shopping-bag" />
+          &nbsp;
+          ADD TO BAG
```

Comment:
instead of having the whole button conditionally render, can just make this text conditionally render